workflow/scripts/prepare_capacityfactors_point_layout.py

```diff
@@ -0,0 +1,72 @@
+"""Prepare PV capacityfactors, given a cutout, a layout, spatial units to aggregate to and technology specifications."""
```

Comment:
This seems to share a lot of code with `prepare_capacityfactors_raster_layout.py` so I would merge this into one